src/main/java/teammates/storage/entity/AccountRequest.java

```diff
@@ -0,0 +1,100 @@
+package teammates.storage.entity;
+
+import java.time.Instant;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Translate;
+
+/**
+ * Represents an AccountRequest entity.
+ */
+@Entity
+@Index
+public class AccountRequest extends BaseEntity {
+
+    @Id
+    private String email;
```

Comment:
Not wise for this to be primary key. A user can request multiple accounts for different institutes.

Reply:
Change primary key to take into account both email and institute